### ISSUE TITLE
About fps

### ISSUE THREAD

**Issue by hcrisA** [song2yu/Mono2Stereo#9]
Hello, may I ask if you could provide the FPS  information for the training set (or the subset mentioned here  #7  ) and the test set? This is very important to me. I would be very grateful if you could offer any assistance.

**Comment by song2yu** [comment]
The exact FPS numbers are a bit uncertain at the moment. I’d recommend downloading a movie from that [site](https://www.3donlinefilms.com/) and checking its FPS directly. Based on our earlier checks, aside from a few titles with unusually high FPS, most of them are fairly consistent.
